index.ftd

```diff
@@ -25,7 +66,30 @@ width: fill
 
 --- cu.buttons-second-row:
 
---- ftd.column:
+--- cu.USS-body:
+
+--- cu.toasts:
+
+--- cu.normal-buttons:
+
+--- cu.hover-buttons:
+
+--- cu.pressed-buttons:
+
+--- cu.focused-buttons:
+
+--- cu.shadow-box:
```

Comment:
pass the color of type `fpm.color-data` to all the component and use it, instead of `fpm.color`